Kitodo/src/main/java/org/kitodo/production/services/data/TaskService.java

```diff
@@ -293,7 +293,7 @@ public TaskDTO convertJSONObjectToDTO(Map<String, Object> jsonObject, boolean re
          * never-ending loops as the list of templates reads the list of tasks.
          */
         int process = TaskTypeField.PROCESS_ID.getIntValue(jsonObject);
-        if (process > 0 && !related) {
+        if (Objects.nonNull(process) && process > 0 && !related) {
```

Comment:
`int` cannot become `null`. Why do you we need the null check here?

Reply:
This is true indeed. I have no idea why I added it.